src/android/FirebaseMessagingPlugin.java

```diff
@@ -220,7 +220,13 @@ private void findChannel(String channelId, CallbackContext callbackContext) thro
             callbackContext.success(new JSONObject()
```

Comment:
It'd be nice to extract new static method
```java
private static JSONObject toJSON(NotificationChannel channel) throws JSONException {
```
And use it 2 times to DRY code.

Reply:
Good suggestion. I've changed it and will push an update shortly.